### PR TITLE
README: Update end-to-end test section on kind

### PR DESCRIPTION
**Pull request by slashpai**
README: Update end-to-end test section on kind

Update image names and example kubernetes version

Signed-off-by: Jayapriya Pai <slashpai9@gmail.com>

## Description

_Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.
If it fixes a bug or resolves a feature request, be sure to link to that issue._

## Type of change

_What type of changes does your code introduce to the Prometheus operator? Put an `x` in the box that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [x] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Changelog entry

_Please put a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note

```

### CODE REVIEW

**Review by fpetkovski — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -174,11 +174,15 @@ kubectl delete --ignore-not-found customresourcedefinitions \
...
+1. `kind create cluster --image=kindest/node:<latest>`. e.g `v1.23.0` version.
 2. `kubectl cluster-info --context kind-kind`. kind version >= 0.6.x
 3. `make image` - build Prometheus Operator docker image locally.
-4. `for n in "operator" "config-reloader" "admission-webhook"; do kind load docker-image "quay.io/prometheus-operator/prometheus-$n:$(git rev-parse --short HEAD)"; done` - publish
-   built locally images to be accessible inside kind.
+4. publish locally built images to be accessible inside kind
+
+    ```bash
+     for n in "operator" "config-reloader"; do kind load docker-image "quay.io/prometheus-operator/prometheus-$n:$(git rev-parse --short HEAD)"; done;
+     kind load docker-image "quay.io/prometheus-operator/admission-webhook:$(git rev-parse --short HEAD)"
```
Comment (simonpasquier) [inline README.md:184]: (nit) reduce repetition a bit :)

```suggestion
     for n in "prometheus-operator" "prometheus-config-reloader" "admission-webhook"; do kind load docker-image "quay.io/prometheus-operator/$n:$(git rev-parse --short HEAD)"; done;
```

Comment (slashpai) [reply README.md]: nice I didn't think of that :)
updated

**Comment by slashpai** [comment]
@simonpasquier this is updated

**Review by simonpasquier — APPROVED** [review]

**Comment by simonpasquier** [comment]
thanks!

**Pull request merged**